src/transformers/models/detr/modeling_detr.py

```diff
@@ -731,8 +732,8 @@ def forward(
             attention_mask (`torch.FloatTensor`): attention mask of size
                 `(batch, 1, target_len, source_len)` where padding elements are indicated by very large negative
                 values.
-            position_embeddings (`torch.FloatTensor`, *optional*):
-                position embeddings that are added to the queries and keys
+            object_queries (`torch.FloatTensor`, *optional*):
+                object_queries that are added to the queries and keys
```

Comment:
```suggestion
                Object_queries that are added to the hidden states.
```
Object queries are actually only added to the hidden states which are the queries.